src/features/left-sidebar/LeftSidebarLinkCallout.js

```diff
@@ -10,8 +10,10 @@ import type { Callout } from './Callout';
 import './styles/LeftSidebarLinkCallout.scss';
 
 type Props = {
+    attachmentPosition?: string,
```

Comment:
would it be possible to type the available positions for both of these props?

Reply:
we actually have them typed here: `src/common/tether-positions.js.flow` i just need to see if that is compatible to import (the "real" constants have been migrated to typescript already).